### DISCUSSION TITLE
incomplete localization

### DISCUSSION THREAD

**Discussion by jackadam1981** [requarks/wiki#6618] [Help / Questions] [answered]
editor selection
![image](https://github.com/requarks/wiki/assets/28656671/fe939083-f6c7-4fc7-8298-2caf2626cdc2)
visual editor
![image](https://github.com/requarks/wiki/assets/28656671/c54c2b0a-538d-493e-9d41-a48805f1037d)
I have loaded the localization files correctly, but found that the editor selection interface is not localized, and the tooltip of the Visual Editor editor is not localized.
After a simple query, I used the secondary development based on ckeditor5, which mainly added the function of insert link to page. The development of ckeditor in recent years is also good, and a customized mode such as online-builder was also born.
Should we implant the insert link to page function as a plug-in (patch)? This will make it easier for everyone to use the onlie-builder ckeditor according to their needs.

The new ckeditor has realized that clipboard pictures can be directly pasted into base64 format, but wiki.js can only paste pictures in word files.

Using the original or online-builder ckeditor will make the wiki.js editor keep up with the development of new technologies and use the more advanced Visual Editor editor.

My requirement is only a hint for localizing the Visual Editor editor.
Maybe you guys have a better solution for editor localization.

Sorry for my bad English, this is all translated by google.

**Comment by jackadam1981** [comment]
The incomplete localization I found was caused by ckediter. Although I can understand the function of the icon at a glance, the education level of end users is beyond our control, so they prefer to use the visual editor, but the visual editor Incomplete localization really prevents them from happily using wiki.js.

**Comment by NGPixel** [comment, marked as answer]
This is being addressed in v3.

v2 has many areas that are not localized.
